lib/foods.js

```diff
@@ -6,7 +6,7 @@ function clearErrors() {
 function checkInputs() {
   clearErrors();
   if ( $('#name-field input').val() == '') {
-    $('#name-field .validation-error').html('Please Enter a Name');
```

Comment:
@lucyconklin Great job with matching up to the specific spelling as defined within the Pivotal ticket